core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -172,6 +172,11 @@ oppia.controller('ExplorationEditor', [
     var _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewTab';
     var _ID_TUTORIAL_SAVE_BUTTON = '#tutorialSaveButton';
 
+    var screenWidth = windowDimensionsService.getWidth();
+    if (screenWidth <= 1280) {
+      _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewSmall';
```

Comment:
@seanlip I think in future, windowWidth might be required for other components. So I created a rootScope varible windowWidth which can be shared across all controllers and dynamically updated depending on the window size. So, @disissid can accordingly change his code to use windowWidth variable of rootScope instead calling $window.resize()